components/sandbox/tiers/production/appstudio/kustomization.yaml

```diff
@@ -13,16 +13,19 @@ resources:
 - tiertemplate-appstudio-clusterresources-593233715-593233715.yaml
 - tiertemplate-appstudio-clusterresources-809836689-809836689.yaml
 - tiertemplate-appstudio-contributor-1817914940-1817914940.yaml
+- tiertemplate-appstudio-contributor-3265408548-3265408548.yaml
```

Comment:
Why do we need such changes?

Reply:
Sorry, it looks really messy, but I'm not aware of anything I can do about it. 

The source of truth and the only thing I'm changing is in [components/sandbox/tiers/src/appstudio](https://github.com/redhat-appstudio/infra-deployments/tree/main/components/sandbox/tiers/src/appstudio). 
We need such changes, because there are references to jvm-build-service and I'm removing them, see an example [here](https://github.com/redhat-appstudio/infra-deployments/pull/5132/commits/6c6aed66ba1035c64fb86d42d9894b501fab424e#diff-94de3b39ce770e418903c21543b146f7dca0e63b16db7af023aec3c63c9e6ed5L107-L115)

After changes are made in the directory, a command is ran as described in the [README](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/sandbox/tiers/README.adoc) which autogenerates majority of the changes in https://github.com/redhat-appstudio/infra-deployments/pull/5132/commits/6c6aed66ba1035c64fb86d42d9894b501fab424e.

This seems to me to indeed be the intended workflow, see https://github.com/redhat-appstudio/infra-deployments/pull/5137 as an example

Reply:
This is normal. Changing one access in a role will case a generation of a new role file, thus explaining all those new lines. This is how kubesaw works to deploy the new tier.